app/controllers/api/subcollections/snapshots.rb

```diff
@@ -2,7 +2,7 @@ module Api
   module Subcollections
     module Snapshots
       def snapshots_query_resource(object)
-        object.snapshots
+        object.cloud_volume_snapshots
```

Comment:
```
cp app/controllers/api/subcollections/snapshots.rb app/controllers/api/subcollections/cloud_volume_snapshots.rb
git checkout master app/controllers/api/subcollections/snapshots.rb # revert
```

So there will be a cloud volume snapshots subcollection that deals with cloud volume snapshots, and the original (untouched) snapshots controller that deals with vm snapshots

Reply:
Hi @kbrock! Thank you so much for the help! I keep getting this error in my local environment when running test cases. Do you know where this error might coming from: 
<img width="1142" alt="Screen Shot 2021-12-15 at 4 41 27 PM" src="https://user-images.githubusercontent.com/82469658/146269315-f246eb77-7e04-45ba-94fd-b4c6120ff2d5.png">